RussianFilter/sections/specific.txt

```diff
@@ -27,6 +27,7 @@ losst.ru###sidebar > .widget_execphp
 gp.by##.article-banner
 gp.by##.article-page > div.content-left > a > img[style="max-width:100%"]
 kinoxa.biz##.f-adv
+editorsdepot.com##.footer-sponsor-section
```

Comment:
I think that it should be added to Base/English filter :)

By the way, there is one more not blocked ad - https://editorsdepot.com/category/?sort=latest
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/62363757-2ff9ed00-b520-11e9-8088-7c88dac96181.png)

</details>

For example:
`||editorsdepot.com/images/edmdbanner.png`

Reply:
@AdamWr,
done: moved to base filter and added one more rule